PCbuild/pythoncore.vcxproj

```diff
@@ -416,9 +416,10 @@
   <ItemGroup>
     <ClCompile Include="..\Modules\_abc.c" />
     <ClCompile Include="..\Modules\_bisectmodule.c" />
-    <ClCompile Include="..\Modules\_blake2\blake2module.c" />
-    <ClCompile Include="..\Modules\_blake2\blake2b_impl.c" />
-    <ClCompile Include="..\Modules\_blake2\blake2s_impl.c" />
+    <ClCompile Include="..\Modules\blake2module.c">
+      <PreprocessorDefinitions Condition="'$(Platform)'!='ARM64'">HACL_CAN_COMPILE_SIMD128;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I'd suggest making these conditions opt-in rather than opt-out, e.g. `'$(Platform)' == 'Win32' or '$(Platform)' == 'x64'`, since these are specific to those two platforms. Also, it looks like there are platforms defined for ARM (32), so just excluding ARM64 is insufficient.

Reply:
Thanks this is now done.